lib/std/Random.zig

```diff
@@ -297,6 +296,7 @@ pub fn float(r: Random, comptime T: type) T {
                     const addl_rand_lz = @clz(r.int(u64));
                     rand_lz += addl_rand_lz;
                     if (addl_rand_lz != 64) {
+                        @brancHint(.likely);
```

Comment:
This looks like a minor typo; shouldn't it be `@branchHint`, not `@brancHint`?

Reply:
Hah, thanks for spotting that. I’m blind.

Reply:
You're all good! Glad I could help! 😁